source/docs/software/basic-programming/reading-stacktraces.rst

```diff
@@ -547,3 +547,33 @@ In the example, the left motor controllers are plugged into PWM ports ``0`` and
                frc::PWMVictorSPX m_rearLeftMotor{1};
 
             };
+
+gradlew is not recognized...
```

Comment:
I don't think this should go in the stack trace documentation. Perhaps a different "general troubleshooting" document?

Reply:
Would you be open to creating this document? I would not fill satisfactory creating a document with less than 20 lines of content.

Reply:
Maybe this can be put with the VS Code C++ Intellisense issue? I think it's the `Known Issues` section.

Reply:
That is an idea. Although, in the past we have only used the known issues article for things that are system wide/bugs that are unfixable. This would be categorized as a user error.